frontend/pages/reports/label-generator.vue

```diff
@@ -319,135 +257,91 @@
       <h1>Homebox Label Generator</h1>
```

Comment:
Translation keys need to be added for all the text on this page but I think thats should be sorted in a seperate PR?